tools/api-markdown-documenter/api-report/api-markdown-documenter.alpha.api.md

```diff
@@ -824,12 +824,9 @@ export class SectionNode extends DocumentationParentNodeBase<SectionContent> {
 // @public
 function shouldItemBeIncluded(apiItem: ApiItem, config: ApiItemTransformationConfiguration): boolean;
 
-// @public @sealed
-export type SpanContent = PhrasingContent | BlockContent;
```

Comment:
Note: this type was added since the last released version, so removing this does not constitute a breaking change.